app/helpers/filter_navigation/people.rb

```diff
@@ -63,16 +63,19 @@ def init_labels
 
     def init_kind_items
       @kind_filter_names.each do |kind, name|
-        types = group.role_types.select { |t| t.kind == kind }
-        next unless visible_role_types?(types)
+        role_types = filter_role_types(kind)
+        next unless visible_role_types?(role_types)
 
-        count = group.people.where(roles: { type: types.collect(&:sti_name) }).distinct.count
-        path = kind == :member ? path() : fixed_types_path(name, types)
-
-        item(name, path, count)
+        count = count_roles(role_types)
+        path = kind == :member ? path() : fixed_types_path(name, role_types)
+        item(name, path, count) unless skip_kind?(kind, count)
       end
     end
 
+    def filter_role_types(kind)
+      (group.role_types + [FutureRole]).select { |t| t.kind == kind }
```

Comment:
Ich verstehe nicht was hier passiert. In `filter_role_types` fügst du `FutureRole` manuell in die Liste hinzu, aber nachher bei den Items nimmst du sie wieder raus via `skip_kind?`